src/main/scala/io/github/pauljamescleary/petstore/config/PetStoreConfig.scala

```diff
@@ -4,7 +4,8 @@ import cats.effect.Sync
 import cats.implicits._
 import pureconfig.error.ConfigReaderException
 
-case class PetStoreConfig(db: DatabaseConfig)
+final case class ServerConfig(host : String, port : Int)
+final case class PetStoreConfig(db: DatabaseConfig, server : ServerConfig)
```

Comment:
I'm super inconsistent with the whitespace on these `:`. I've introduced this inconsistency everywhere in this project so it's fine...